test/repo-init-integration/expected/ci-operator/jobs/org/other/org-other-nonstandard-presubmits.yaml

```diff
@@ -4,7 +4,7 @@ presubmits:
     always_run: true
     branches:
     - nonstandard
-    cluster: api.ci
+    cluster: ci/api-build01-ci-devcluster-openshift-com:6443
```

Comment:
Why did a `ci-operator` test change?

Reply:
Because we are testing the config in the integration test.
The config tells to run that job in build01.

https://github.com/openshift/ci-tools/blob/0fead13f0eb9f78c23e3fff1c7042b9d345421c4/test/repo-init-integration/input/core-services/sanitize-prow-jobs/_config.yaml#L7-L10